code/drasil-printers/lib/Language/Drasil/Debug/Print.hs

```diff
@@ -282,50 +247,3 @@ renderUsedUIDs chs = header (text "UIDs" $$ nest 40 (text "Associated Chunks"))
   $$ vcat (map renderUsedUID chs)
   where
     renderUsedUID (u, chks) = text (show u) $$ nest 40 (text chks)
-
--- | For the last section of the log output. Shows which chunk UID is being used at which stage.
--- Note that chunks used at a "higher stage" (like 'Concept's and 'QuantityDict's) will still be built off of the
--- more basic types (like 'IdeaDict's), they are just not explicitly used in that manner.
--- Also, some chunks may have been "downgraded" when put into the database (for example, mapping a
--- 'QuantityDict' wrapper onto things like Constrained and Unital chunks happens often).
-mkListShowUsedUIDs :: PrintingInformation -> [(UID, String)]
-mkListShowUsedUIDs PI { _ckdb = db } = sortBy (compare `on` fst)
```

Comment:
Similar logic. Except now that we're moving away from nested chunks, this won't be relevant.